packages/salesforcedx-vscode-apex-debugger/package.nls.json

```diff
@@ -1,9 +1,15 @@
 {
-  "user_id_filter_text": "A comma-separated list of user IDs with requests whitelisted for debugging",
-  "request_type_filter_text": "A comma-separated list of ApexRequesType values whitelisted for debugging",
-  "entry_point_filter_text": "A regular expression matching entry points whitelisted for debugging",
+  "user_id_filter_text":
+    "A comma-separated list of user IDs with requests whitelisted for debugging",
+  "request_type_filter_text":
+    "A comma-separated list of ApexRequesType values whitelisted for debugging",
+  "entry_point_filter_text":
+    "A regular expression matching entry points whitelisted for debugging",
   "sfdx_project_text": "Location of SFDX project",
   "launch_snippet_label_text": "Apex Debugger: Launch",
-  "launch_snippet_description_text": "A new configuration for launching Apex Debugger",
-  "launch_snippet_name": "Launch Apex Debugger"
+  "launch_snippet_description_text":
+    "A new configuration for launching Apex Debugger",
+  "launch_snippet_name": "Launch Apex Debugger",
+  "trace_text":
+    "Produce diagnostic output. Instead of setting this to true you can list one or more selectors separated with commas. The 'all' selector enables very detailed output."
```

Comment:
Include possible values so people know what to put in.

Reply:
@DatGuyJonathan Should we add more logging to existing areas and add them here as well?

Reply:
It looks like you mainly trace variable handling for now. You mentioned 'dap' somewhere in there; I think that could be used to trace the order of initializeRequest, attachRequest, launchRequest, setBreakPointsRequest, etc. We can also trace debugger events in handleEvent(). Sometimes we disconnect and reconnect from streaming api, so we can log that as well.

Reply:
`'protocol'` (former `'dap'`) is used to enable tracing of the communication between VS Code and the debug adapter.